virt/virt-4-12-release-notes.adoc

```diff
@@ -100,9 +100,11 @@ The SVVP Certification applies to:
 === Installation
 
 
-//[id="virt-4-12-networking-new"]
-//=== Networking
+[id="virt-4-12-networking-new"]
+=== Networking
 
+//CNV-18349
+* You can now xref:../virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc#virt-about-services_virt-creating-service-vm[configure a load balancing service] by using the MetalLB Operator in layer 2 mode.
```

Comment:
IMO, you can also add link to the MetalLB Operator with this --> https://54300--docspreview.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-operator-install.html#metallb-operator-install

Reply:
@sbulage The documentation that the release note links to includes links to the topics "Installing the MetalLB Operator" and "Configuring services to use MetalLB" in the _Additional resources_ section.

Reply:
Okay, we could highlight it more specifically. Or you can add Note, mentioning that if MetalLB is not installed here you can find links in additional resources section.

Reply:
@sbulage I have linked to the "Installing the MetalLB Operator" topic from the release note per your earlier suggestion. Can you please review and approve? Thanks!